tests/integration/cqlengine/query/test_named.py

```diff
@@ -259,3 +263,95 @@ def test_get_multipleobjects_exception(self):
             self.table.objects.get(test_id=1)
 
 
+class TestQuerySetCountSelectionAndIteration(BasicSharedKeyspaceUnitTestCase):
+
+    def test_named_table_with_mv(self):
+        """
+        Test NamedTable access to materialized views
+
+        Creates some materialized views using Traditional CQL. Then ensures we can access those materialized view using
+        the NamedKeyspace, and NamedTable interfaces. Tests basic filtering as well.
+
+        @since 3.0.0
+        @jira_ticket PYTHON-406
+        @expected_result Named Tables should have access to materialized views
+
+        @test_category materialized_view
+        """
+        connection.setup(['127.0.0.1'], self.keyspace_name)
+
+        # Create a base table and two materialized views
+        create_table = """CREATE TABLE {0}.scores(
+                        user TEXT,
+                        game TEXT,
+                        year INT,
+                        month INT,
+                        day INT,
+                        score INT,
+                        PRIMARY KEY (user, game, year, month, day)
+                        )""".format(self.keyspace_name)
+
+        self.session.execute(create_table)
+        create_mv = """CREATE MATERIALIZED VIEW {0}.monthlyhigh AS
+                        SELECT game, year, month, score, user, day FROM {0}.scores
+                        WHERE game IS NOT NULL AND year IS NOT NULL AND month IS NOT NULL AND score IS NOT NULL AND user IS NOT NULL AND day IS NOT NULL
+                        PRIMARY KEY ((game, year, month), score, user, day)
+                        WITH CLUSTERING ORDER BY (score DESC, user ASC, day ASC)""".format(self.keyspace_name)
+
+        self.session.execute(create_mv)
+
+        create_mv_alltime = """CREATE MATERIALIZED VIEW {0}.alltimehigh AS
+                        SELECT * FROM {0}.scores
+                        WHERE game IS NOT NULL AND score IS NOT NULL AND user IS NOT NULL AND year IS NOT NULL AND month IS NOT NULL AND day IS NOT NULL
+                        PRIMARY KEY (game, score, user, year, month, day)
+                        WITH CLUSTERING ORDER BY (score DESC)""".format(self.keyspace_name)
+
+        self.session.execute(create_mv_alltime)
+
+        # Populate the base table with data
+        prepared_insert = self.session.prepare("""INSERT INTO {0}.scores (user, game, year, month, day, score) VALUES  (?, ?, ? ,? ,?, ?)""".format(self.keyspace_name))
```

Comment:
This block could be tighter and more efficient using the prepared statement and a list of args in  [execute_concurrent_with_args](http://datastax.github.io/python-driver/api/cassandra/concurrent.html#cassandra.concurrent.execute_concurrent_with_args)